src/datasets/blackspots/models.py

```diff
@@ -19,6 +19,10 @@ class SpotType(DjangoChoices):
         protocol_ernstig = ChoiceItem()
         protocol_dodelijk = ChoiceItem()
         risico = ChoiceItem()
+        gebiedslocatie_ivm = ChoiceItem()
```

Comment:
add a comment to explain what ivm is ("infrastructurele verkeersveiligheidsmaatregel")